tests/components/deconz/test_init.py

```diff
@@ -71,26 +75,27 @@ async def test_setup_entry_multiple_gateways(hass):
 
 async def test_unload_entry(hass):
     """Test being able to unload an entry."""
-    gateway = await setup_deconz_integration(hass)
+    config_entry = await setup_deconz_integration(hass)
     assert hass.data[deconz.DOMAIN]
 
-    assert await deconz.async_unload_entry(hass, gateway.config_entry)
+    assert await deconz.async_unload_entry(hass, config_entry)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)